src/setup.ts

```diff
@@ -32,6 +33,17 @@ export async function setupDocs(dir: string) {
     routeRules: {
       ...Object.fromEntries(Object.entries(config.redirects || {}).map(([from, to]) => [from, { redirect: to }])),
     },
+    tailwindcss: {
+      config: {
+        theme: {
+          extend: {
+            colors: {
+              theme: getColors(config.theme || ''),
```

Comment:
I guess we need to use a default value for it as fallback no? (what happens if user does not set `theme`)

Reply:
Yes, we can use the yellow as a fallback